woocommerce/woocommerce
# Update CI config to run tests on changes of i18n dir

### Changes proposed in this Pull Request

Include the `i18n/**/*.php` path in the changes that trigger e2e and api tests in CI. 
https://github.com/woocommerce/woocommerce/pull/51959 was recently merged and broke the API tests in trunk because the checks did not run on PR.

Tests will be fixed with https://github.com/woocommerce/woocommerce/pull/52965, which needs to be merged first.

### How to test the changes in this Pull Request:

Create a branch from this one and update something in the `plugins/woocommerce/i18n` folder. Run the ci-jobs util and check the output. Is should include the e2e and api tests.

```pnpm utils ci-jobs```
